build-start-daemon.sh

```diff
@@ -104,6 +105,7 @@ Environment=CEDANA_GPU_CONTROLLER_PATH=$CEDANA_GPU_CONTROLLER_PATH
 Environment=CEDANA_PROFILING_ENABLED=$CEDANA_PROFILING_ENABLED
 Environment=CEDANA_OTEL_ENABLED=$CEDANA_OTEL_ENABLED
 Environment=CEDANA_OTEL_PORT=$CEDANA_OTEL_PORT
+Environment=CEDANA_PORT=$CEDANA_PORT
```

Comment:
we don't pick CEDANA_PORT from env use `--port` instead

Reply:
At the time of making this, I thought we needed this for the controller to pick up the env. But I just realised that the secrets in helm-chart must suffice. If that's right we can close this pr right? because I've added --port in helm-chart templates.

Reply:
--port is required when starting the daemon.

```
cedana daemon start --port ${CEDANA_PORT:8080}
```

Reply:
this need to add at the bottom of build-start-daemon script.

Reply:
I hope this is fixed after the last commit.